db/schema.rb

```diff
@@ -11,15 +11,18 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema.define(version: 20150919025822) do
+ActiveRecord::Schema.define(version: 20151026180945) do
```

Comment:
The conflicts we were having with the version have to do with it not being up-to-date with penzance/master. Once you've merged in penzance/master changes (and run `bundle exec rake db:migrate`) this version stuff will be fixed I think.